lib/controller/api_catalog.es6.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+require('../component/catalog_table.es6.js');
+require('../component/api_analytics.es6.js');
+
+angular.module('confluence')
+  .controller('catalogController',
+    ['$scope', 'api', function($scope, api) {
```

Comment:
Why array syntax in some places and bare functions in others? If you want it to be minification-safe, then you should use the array style everywhere. Otherwise the simple form suffices.